pkgs/applications/virtualization/OVMF/default.nix

```diff
@@ -14,14 +14,36 @@ in
 stdenv.mkDerivation (edk2.setup "OvmfPkg/OvmfPkg${targetArch}.dsc" {
   name = "OVMF-2014-12-10";
 
-  buildInputs = [nasm iasl];
+  # TODO: properly include openssl for secureBoot
```

Comment:
Does this TODO still apply? It looks like openssl is included if secureBoot is inabled.